deployment/src/main/resources/dev-ui/quinoa/qwc-quinoa-card.js

```diff
@@ -0,0 +1,84 @@
+                                       alt="${NAME}" 
+                                       title="${NAME}"
+                                       width="32" 
+                                       height="32">
+                </div>
+                <div class="description">${DESCRIPTION}</div>
```

Comment:
Why not the description from the attributes ? (so this.description)

Reply:
The description in the YAML is very very long making this widget in the UI too big so I shortened it for clarity and brevity without touching the YML file.